src/wrapper/sdl.zig

```diff
@@ -224,10 +224,16 @@ pub fn init(flags: InitFlags) !void {
         return makeError();
 }
 
+pub const initSubmodule = init;
```

Comment:
[`SDL_InitSubSystem`](https://wiki.libsdl.org/SDL_InitSubSystem) is not `SDL_Init`, even if they are defined identical in SDL2. Please use `SDL_InitSubSystem` here.

Also please call the function alias `initSubSystem`